_src/body/_body.jade

```diff
@@ -1,13 +1,10 @@
 .body__wrapper 
   .body__card
-    .body__info
-      .body__frame
-        img.body__image(src="assets/dogola.png", alt="OOPSS")
-      .body__content
-        h3.body__title DOGOLA
-        p.body__text Beyond the anthills of the dawning of this plague, said I've lost my way. Even if this cul de sac did pay. To reach inside a vault whatever be the cost. Sterling clear. Blackend ice. When they drag the lake theres nothing left at all 
-    .body__collapse
-      span.body__icon
+    .body__frame
+      img.body__image(src="assets/dogola.png", alt="OOPSS")
+    .body__content
+      h3.body__title DOGOLA
+      p.body__text Beyond the anthills of the dawning of this plague, said I've lost my way. Even if this cul de sac did pay. To reach inside a vault whatever be the cost. Sterling clear. Blackend ice. When they drag the lake theres nothing left at all 
   .body__card
```

Comment:
We can size 50px down the body__card element. It should fit better the entire content held in it.